src/gui/sdlmain.cpp

```diff
@@ -3004,15 +3023,17 @@ void Config_Add_SDL() {
 	Pbool->Set_help("Wait before closing the console if dosbox has an error.");
 
 	Pmulti = sdl_sec->Add_multi("priority", Property::Changeable::Always, ",");
-	Pmulti->SetValue("higher,normal");
+	Pmulti->SetValue("auto,auto");
```

Comment:
Given the Windows scheduler needs all the help we can give it, while *nix schedulers are sophisticated and need little help, I think we do something like:

``` c++
#if defined(WIN32)
Pmulti->SetValue("higher, normal");
#else
Pmulti->SetValue("auto, auto");
#endif 
```

Reply:
Wow, what is the CPU on that Windows 10 box?

Reply:
Do you get roughly the same behavior with normal,normal?

Reply:
Yes, `auto,auto` was the same `normal, normal` on Windows - both were fully "stuck" behind the competing load, and Quake didn't make it to its opening screen for roughly 20 minutes.